lib/meilisearch/pagination/kaminari.rb

```diff
@@ -25,12 +25,12 @@ def offset(num)
       class << self
         def create(results, total_hits, options = {})
           offset = ((options[:page] - 1) * options[:per_page])
-          array = new results, :offset => offset, :limit => options[:per_page], :total_count => total_hits
+          array = new results, :limit => options[:per_page], :offset => offset
```

Comment:
Why do we change this file in this PR?

Reply:
Pagination with Kaminari wasn't working, I didn't understand why, so I went through [their doc](https://www.rubydoc.info/gems/kaminari/0.12.3/Kaminari/PaginatableArray) and realized the last parameter wasn't necessary. When I removed it, pagination started working.

Reply:
I've just added `total_count` at the end of the line and it works:
```
array = new results, :limit => options[:per_page], :offset => offset, :total_count => total_hits
```